views/js/qtiCreator/widgets/static/img/states/Active.js

```diff
@@ -232,10 +233,32 @@ define([
                                 img.removeAttr('height');
                             }
 
-                            if($.trim($label.val()) === ''){
-                                label = _extractLabel(file);
-                                img.attr('alt', label);
-                                $label.val(label).trigger('change');
+                            if($.trim($alt.val()) === ''){
+                                if(alt === ''){
+                                    alt = _extractLabel(file);
+                                }
+                                img.attr('alt', alt);
+                                $alt.val(alt).trigger('change');
+                            }
+                            else if(alt !== ''){
```

Comment:
I wonder if the "else" condition is actually relevant ... have you got a reasonable explication for this ?